base/ca/src/main/java/org/dogtagpki/server/ca/v2/EmptyAuthMethodFilter.java

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package org.dogtagpki.server.ca.v2;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebFilter;
+
+import org.dogtagpki.server.v2.AuthMethodFilter;
+
+@WebFilter(servletNames = {"caInfo", "caCert", "caCertRequest"})
+public class EmptyAuthMethodFilter extends AuthMethodFilter {
+
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    public void init() throws ServletException {
+        setAuthMethod("");
```

Comment:
Similar to `ACLFilter`, I'm hoping that an empty `authMethod` (which is the default) will be bypassed so we don't need to define empty filter classes.

Reply:
same as above